R/carehomes.R

```diff
@@ -68,10 +72,15 @@ carehomes_parameters <- function(start_date, region,
                                  react_specificity = 0.99,
                                  react_sensitivity = 0.99,
                                  prop_noncovid_sympt = 0.01,
+                                 rel_susc = 1,
                                  exp_noise = 1e6) {
   ret <- sircovid_parameters_shared(start_date, region,
                                     beta_date, beta_value)
 
+  ## TO DO: we should add some checks for pillar2_specificity,
```

Comment:
Can you either add these or open a ticket that explains the problem more fully and reference it here?